src/app/game/bullet-math/page.tsx

```
@@ -0,0 +1,267 @@
+              value={userAnswer}
+              onChange={(e) => setUserAnswer(e.target.value)}
+              onKeyDown={(e) => { if (e.key === 'Enter') onSubmitAnswer() }}
+              className={`${styles.input} ${isWrong ? styles.wrong : ''}`}
+              placeholder="?"
+            />
+          </>
+        ) : (
+          !gameOver && (
+            <button className={`${styles.btn} ${styles.neutral} ${styles.startBtn}`} onClick={startGame}>
+              Press Start
+            </button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)